backend/probloom/prob/models.py

```diff
@@ -33,7 +31,7 @@ def __str__(self):
 class ProblemSet(Model):
     title = CharField(max_length=100, default="default title")
     date = DateTimeField(auto_now_add=True, blank=True)
```

Comment:
Consider changing this field's name from `date` to something like `created_time`. Since there exists both `DateField` (which stores `datetime.date`) and `DateTimeField` (which stores `datetime.datetime`), it could be confusing if a `DateTimeField` is named `date`.

I think this has low importance, so you don't have to change it if you don't feel like it.